src/Raven.Server/Documents/PeriodicBackup/Restore/Sharding/SingleShardRestoreBackupTask.cs

```diff
@@ -44,7 +40,7 @@ protected override Task OnAfterRestoreAsync()
         {
             SmugglerBase.EnsureProcessed(Result, skipped: false);
             Progress.Invoke(Result.Progress);
-            return Task.CompletedTask;
+            return SaveDatabaseRecordAsync(RestoreSettings.DatabaseRecord.DatabaseName, RestoreSettings.DatabaseRecord, databaseValues: null, Result, Progress);
```

Comment:
why do we need to save the database record here?

Reply:
on import I'm using `onDatabaseRecordAction` ([here](
https://github.com/ravendb/ravendb/blob/v6.0/src/Raven.Server/Documents/PeriodicBackup/Restore/AbstractRestoreBackupTask.cs#L260)) - in order to enable revisions from smuggler record before import, like we do in regular restore.
If we don't save the database record afterwards then it's basically like skipping database-record processing

in RavenDB-19202 we'll make adjustments such that the shard with the most up-to-date database record will be the only one that operates on database record